chore: Typo and grammar fixes (#3)
antfu/contribute: README.md

If it's a Node.js package, it will start the build process in watch mode, or [stub the passive watcher when using `unbuild`](https://antfu.me/posts/publish-esm-and-cjs#stubbing).

> If it's a frontend project, it commonly starts the dev server then you can develop and see the changes in realtime.

Reviewer: More suggestions for this one:

```suggestion
If it's a frontend project, it usually starts the dev server. You can then develop and see the changes in real time.
```

[Corepack](https://nodejs.org/api/corepack.html) makes sure you are using the correct version for package manager when you run corresponding commands. Projects might have `packageManager` field in their `package.json`.

> Under projects with configures shown on the right, when Corepack is enabled, it will install `v7.1.5` of `pnpm` if you don't have it already and use that version to run your command. This makes sure everyone working on this project will have the same behavior for the dependencies and the lockfile.

Reviewer: ```suggestion
Under projects with configuration as shown on the right, when Corepack is enabled, it will install `v7.1.5` of `pnpm` (if you don't have it already) and use it to run your command. This makes sure everyone working on this project will have the same behavior for the dependencies and the lockfile.
```
